core-helm-chart/values.yaml

```diff
@@ -16,20 +16,20 @@
 image:
   repository: quay.io/kubestellar/kubestellar
   pullPolicy: IfNotPresent
-  tag: release-0.14
+  tag: sp-mgt-pr2
```

Comment:
I can not tell what is in that image. The metadata in tags says it was not built from a clean checkout. And about the checkout it was built from, I can not identify it:

```
mspreitz@mjs13 kubestellar % git branch -r --contains 6c2f99c24
error: malformed object name 6c2f99c24
```

Reply:
This image is built using the files in this PR and the `make kubestellar-image` command.  Is there something else we need to do to build the image?

Reply:
Oh, I found that commit (https://github.com/effi-ofer/edge-mc/commit/6c2f99c24645268edee01762c3bcf8716a667d14). It is outdated now. You need to build again with the current contents of the PR's branch. Well, after you rebase and deal with the merge conflicts.